unity-wrapper/src/main/java/com/linecorp/linesdk/unitywrapper/activity/LineSdkWrapperActivity.kt

```diff
@@ -0,0 +1,150 @@
+        startLineSdkLoginActivity()
+    }
+
+    private fun parseIntent() {
+        identifier = intent.getStringExtra(KEY_IDENTIFIER) ?: ""
+        channelId = intent.getStringExtra(KEY_CHANNEL_ID) ?: ""
+        onlyWebLogin = intent.getBooleanExtra(KEY_ONLY_WEB_LOGIN, false)
+
+        val scopeString = intent.getStringExtra(KEY_SCOPE) ?: ""
+        scope = scopeString.split(",")
+            .map { Scope.findScope(it) }
+            .filter { it != null }
+            .map { it -> it!!}
+            .toList()
+    }
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        if (requestCode != REQUEST_CODE_LOGIN) return
+
+        if (resultCode != Activity.RESULT_OK || data == null) {
+            CallbackPayload(identifier, "login error")
+                .sendMessageError()
+        }
+
+        val result = LineLoginApi.getLoginResultFromIntent(data)
+        Log.d(TAG, "login result:$result")
+
+        when(result.responseCode) {
+            LineApiResponseCode.CANCEL -> {
+                Log.d(TAG, "login canceled")
+                val errorForUnity = ErrorForUnity(
+                    result.responseCode.ordinal,
+                    "login is canceled."
+                )
+                CallbackPayload(
+                    identifier,
+                    gson.toJson(
+                        errorForUnity
+                    )
+                ).sendMessageError()
+            }
+
+            LineApiResponseCode.SUCCESS -> {
+                Log.d(TAG, "login success")
+                val resultJsonString = gson.toJson(
+                    LoginResultForUnity.convertLineResult(result))
+                CallbackPayload(
+                    identifier,
+                    resultJsonString
+                ).sendMessageOk()
+            }
+
+            else -> {
+                Log.d(TAG, "login error")
+                val errorForUnity = ErrorForUnity(
+                    result.responseCode.ordinal,
+                    result.errorData.message ?: "login error"
+                )
+                CallbackPayload(
+                    identifier,
+                    gson.toJson(
+                        errorForUnity
+                    )
+                )
+                    .sendMessageError()
+            }
+        }
+
+        finish()
+    }
+
+    private fun startLineSdkLoginActivity() {
+        val lineAuthenticationParams = LineAuthenticationParams.Builder().scopes(scope).build()
+        val loginIntent = if (onlyWebLogin) {
+            LineLoginApi.getLoginIntentWithoutLineAppAuth(
+                this,
+                channelId,
+                lineAuthenticationParams
+            )
+        } else {
+            LineLoginApi.getLoginIntent(
+                this,
+                channelId,
+                lineAuthenticationParams
+            )
+        }
+        startActivityForResult(loginIntent,
+            REQUEST_CODE_LOGIN
+        )
```

Comment:
The length of this line doesn't exceed the max 100 line, we don't need to wrap it.
```kotlin
startActivityForResult(loginIntent, REQUEST_CODE_LOGIN)
```